pants.toml

```diff
@@ -64,6 +64,9 @@ build_ignore.add = [
   "testprojects/src/go/**",
 ]
 
+# Developers can customize Pants settings by using a `pants.dev.toml` file (which is .gitignored)
+pantsrc_files.add = ["pants.dev.toml"]
```

Comment:
Ah, in the OP - you want to have the Pants repo only warn but other repos default to info?

Reply:
Nailed it. They're all related. 

I'll probably put the `--quiet` arg addition in my global pants rc because I don't care about seeing that stuff in any repo.

But in local RC files I have things I want per-repo. E.g. I want to see stats logged for my work monorepo, but not for Pants. And I don't want to blast every dev with those logs either.
In the Pants repo, I'd like to define my own aliases (which will be different, because it uses different tools). Stuff like that.

But. in general, I want to customize options per-repo for my own sake but don't want to dirty the git-tracked files.

Reply:
Ok. makes sense to me.